src/validator.ts

```diff
@@ -3,27 +3,46 @@ import {
   parseMessageWithCustomArgs,
 } from '@lemoncode/fonk';
 
-// TODO: Add validator type
-const VALIDATOR_TYPE = '';
+const VALIDATOR_TYPE = 'ARRAY_REQUIRED';
 
-// TODO: Add default message
-let defaultMessage = '';
+let defaultMessage = 'The value must be an array';
 export const setErrorMessage = message => (defaultMessage = message);
 
+const defaultCustomArgs: CustomValidatorArgs = {
+  minLength: 0,
```

Comment:
Update `minLength` to 1. Because it's like `required` validator for string value, you need at least one char to get valid field.

Reply:
Done!